packages/@react-types/combobox/src/index.d.ts

```diff
@@ -28,13 +28,16 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   /** Handler that is called when the ComboBox input value changes. */
   onInputChange?: (value: string) => void,
   /** Whether the ComboBox allows a non-item matching input value to be set. */
-  allowsCustomValue?: boolean,
+  allowsCustomValue?: boolean
   // /**
   //  * Whether the Combobox should only suggest matching options or autocomplete the field with the nearest matching option.
   //  * @default 'suggest'
   //  */
   // completionMode?: 'suggest' | 'complete',
-  /**
+}
+
+export interface AriaComboBoxProps<T> extends ComboBoxProps<T>, DOMProps {
```

Comment:
this interface should probably include the other parts (all the refs) defined in useComboBox